sls-cronjob/serverless.yml

```diff
@@ -0,0 +1,37 @@
+service: sls-cronjob
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: ${opt:region, "us-west-2"}
+  stage: ${opt:staging, "dev"}
+  profile: ${opt:profile, "serverlessguru-internal"}
+  environment:
+    SLS_DEBUG: "*"
+    REGION: ${self:provider.region}
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - ses:SendEmail
+        - ses:SendTemplatedEmail
+      Resource: arn:aws:ses:#{AWS::Region}:#{AWS::AccountId}:identity/nadtakan@serverlessguru.com
+    - Effect: Allow
+      Action:
+        - cloudformation:DescribeStackResourceDrifts
+      Resource: arn:aws:cloudformation:#{AWS::Region}:#{AWS::AccountId}:*
+
+plugins:
+  - serverless-pseudo-parameters
```

Comment:
Add serverless-webpack plugin, https://github.com/serverless-guru/templates/tree/master/sls-webpack/01_webpack

We don't need the copy-plugin as there aren't additional files to be packaged, webpack.config.js should look like this, https://github.com/serverless-guru/templates/tree/master/sls-webpack/01_webpack#create-webpackconfigjs-file